presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -55,6 +55,7 @@ statement
         ('(' explainOption (',' explainOption)* ')')? statement        #explain
     | SHOW TABLES ((FROM | IN) qualifiedName)? (LIKE pattern=STRING)?  #showTables
     | SHOW SCHEMAS ((FROM | IN) identifier)?                           #showSchemas
+    | SHOW CREATE TABLE qualifiedName                                  #showCreateTable
```

Comment:
Let's move this to before "SHOW TABLES", since it makes sense to keep "SHOW TABLES / SCHEMAS / CATALOGS" together.